index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/67e6112a71.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" defer src="action.js"></script>
+    <title>Document</title>
+</head>
+<body>
+    <div class="sign-in"><i class="fa fa-facebook-f" ></i><i class="fa-brands fa-twitter"></i> <p>English</p> <p>MyPage</p> <p>logout</p></div>
+    <nav  id="menu" class="nav-bar-i"><i id="menu-b" onclick="changeIcon();myFunction()" class="fa-sharp fa-solid fa-bars"></i>
+          <ul class="nav-bar-courtain">
+            <li><a href="about.html">About</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+        </ul>
+        <div class="nav-links">
+            <div class="logo-nav"><a href="index.html"><img src="./earth-logo.png"></a></div>
+        <a href="about.html" class="about">About</a>
+        <div>Program</div>
+        <div>Join</div>
+        <div>Sponsor</div>
+        <div>News</div>
+        <div class="compaign">Rescue the Earth Compaign</div>
+    </div></nav>
+    <section class="home-page">
+    <div class="title">
+        <div class="greeting">"Hello"</div>
+        <h1 class="home-title">Together to rescue the earth</h1>
```

Comment:
- [ ] Kindly change this title to `uppercase` because, in the original design, the title is uppercase and increase its font a bit. And instead of having it in one, I would suggest you break it into two lines for example. Like this 👇️

`<h1 class="home-title">Together to rescue <br> the earth</h1>`